install/InstallTools.py

```diff
@@ -4910,31 +5005,35 @@ def indocker():
     @staticmethod
     def init(name=None):
         if not DockerFactory._init:
-            rc, out, _ = Tools.execute("cat /proc/1/cgroup", die=False, showout=False)
-            if rc == 0 and out.find("/docker/") != -1:
-                # nothing to do we are in docker already
-                return
-
-            MyEnv.init()
-
-            if MyEnv.platform() == "linux" and not Tools.cmd_installed("docker"):
-                UbuntuInstaller.docker_install()
-                MyEnv._cmd_installed["docker"] = shutil.which("docker")
-
-            if not Tools.cmd_installed("docker"):
-                raise Tools.exceptions.Operations("Could not find Docker installed")
+            if not MyEnv.platform_is_windows:
+                rc, out, _ = Tools.execute("cat /proc/1/cgroup", die=False, showout=False)
+                if rc == 0 and out.find("/docker/") != -1:
+                    # nothing to do we are in docker already
+                    return
 
-            DockerFactory._init = True
-            cdir = Tools.text_replace("{DIR_BASE}/var/containers")
-            Tools.dir_ensure(cdir)
-            for name_found in os.listdir(cdir):
-                if not os.path.isdir(os.path.join(cdir, name_found)):
-                    # https://github.com/threefoldtech/jumpscaleX_core/issues/297
-                    # in case .DS_Store is created when opened in finder
-                    continue
-                # to make sure there is no recursive behaviour if called from a docker container
-                if name_found != name and name_found.strip().lower() not in ["shared"]:
-                    DockerContainer(name_found)
+                MyEnv.init()
+
+                if MyEnv.platform() == "linux" and not Tools.cmd_installed("docker"):
+                    UbuntuInstaller.docker_install()
+                    MyEnv._cmd_installed["docker"] = shutil.which("docker")
+
+                if not Tools.cmd_installed("docker"):
+                    raise Tools.exceptions.Operations("Could not find Docker installed")
+
+                DockerFactory._init = True
+                cdir = Tools.text_replace("{DIR_BASE}/var/containers")
+                Tools.dir_ensure(cdir)
+                for name_found in os.listdir(cdir):
+                    if not os.path.isdir(os.path.join(cdir, name_found)):
+                        # https://github.com/threefoldtech/jumpscaleX_core/issues/297
+                        # in case .DS_Store is created when opened in finder
+                        continue
+                    # to make sure there is no recursive behaviour if called from a docker container
+                    if name_found != name and name_found.strip().lower() not in ["shared"]:
+                        DockerContainer(name_found)
+            else:
+                MyEnv.init()
```

Comment:
do a check if docker is isntalled otherwise raise and point to docker install manual